<launch>
  <node respawn="true" pkg="joy" type="joy_node" name="joy">
    <param name="dev" value="/dev/input/js0" />
    <param name="deadzone" value="0.05" />
  </node>

	<node pkg="darpasubt" type="joystick_teleop" name="joystick_teleop" output="screen">
    <param name="frequency" value="10"/>
    <param name="axis_linear" value="1"/>
    <param name="axis_angular" value="2"/>
    <param name="scale_linear" value="2"/>
    <param name="scale_angular" value="0.5"/>
    <remap from="/cmd_vel" to="/X1_1/cmd_vel"/>
  </node>
</launch>
